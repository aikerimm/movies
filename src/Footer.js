import React from 'react'

export default function Footer () {
  return React.createElement('footer', { className: 'footer' },
    React.createElement('small', null, 'Â© 2022 Aigerim'))
}
