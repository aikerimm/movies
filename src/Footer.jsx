import React from 'react';

const Footer = () => React.createElement('footer', {className: 'footer'},
  React.createElement('small', null, 'Â© 2022 Aigerim'));

export default Footer; 
