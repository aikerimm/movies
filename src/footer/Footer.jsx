import React from 'react';
import './footer.css';

const Footer = () => (
  <footer className='footer'>
    <b>netflix</b>roulette
  </footer>
);

export default Footer;
