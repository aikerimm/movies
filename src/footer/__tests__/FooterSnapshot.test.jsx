import renderer from 'react-test-renderer';
import Footer from '../Footer.jsx';

it('Footer renders correctly', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});