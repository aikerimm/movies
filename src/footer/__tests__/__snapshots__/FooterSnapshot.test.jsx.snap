// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <b>
    netflix
  </b>
  roulette
</footer>
`;
